### ISSUE TITLE
Fluxstore Multivendor 1.7.7 payment error

### ISSUE THREAD

**Issue by bangbiggy** [inspireui/support#4677]
Hello, am getting the follow error when I reach the payment tab.  
![Screenshot_20200820-094656](https://user-images.githubusercontent.com/60167322/90726764-eca92280-e2ca-11ea-9f94-15316d7680d3.png)
I Only Have One Payment Method on my website which is cash on delivery. And work fine on the website. 

Proof of purchase:
![Screenshot_20200820-094211](https://user-images.githubusercontent.com/60167322/90726920-2712bf80-e2cb-11ea-85c7-f8b66cbd482e.png)

Please help asap!! 
Thanks.

**Comment by minhcasi** [comment]
@bangbiggy - could i know which the plugins you are using (Dokan or WCFM)? 

Please give a try to test with the default website config on this link https://docs.inspireui.com/fluxstore/multi-vendor/#1-website-integration - thank you!

**Comment by eraldix** [comment]
Same problem here with cash on delivery on WCFM

**Comment by eraldix** [comment]
Hey didd you fix this issue

**Comment by bangbiggy** [comment]
> Hey didd you fix this issue

yes.. for me the problem was on the  website when i added a rule on cod... i remove the rule and fix the problem. although wish the author could fix the app so it takes the rules too.

**Comment by eraldix** [comment]
What rule, can you be more specific please, I'm having trouble with it

**Comment by bangbiggy** [comment]
> What rule, can you be more specific please, I'm having trouble with it

![ss](https://user-images.githubusercontent.com/60167322/91784655-16881080-ec0c-11ea-80ee-197cff52be57.jpg)
i left the "enable for shipping method" empty.

**Comment by eraldix** [comment]
> > What rule, can you be more specific please, I'm having trouble with it
> 
> ![ss](https://user-images.githubusercontent.com/60167322/91784655-16881080-ec0c-11ea-80ee-197cff52be57.jpg)
> i left the "enable for shipping method" empty.

It fixed for me but it doesn't get the shipping , shipping goes 0.

**Comment by bangbiggy** [comment]
> It fixed for me but it doesn't get the shipping , shipping goes 0.
how dd you setup your shipping?

**Comment by eraldix** [comment]
Same issue
